### PR TITLE
Added tlg0317.tlg002.1st1K-grc1.xml

### PR DESCRIPTION
**Pull request by sonofmun**
Acta Joannis (recensio)

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +35 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +2529 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0317/__cts__.xml`                          | New      |
| `./data/tlg0317/tlg002/__cts__.xml`                   | New      |
| `./data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml` | New      |
| `./data/tlg0317/__cts__.xml`                          | Passing  |
| `./data/tlg0317/tlg002/__cts__.xml`                   | Passing  |
| `./data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3631)*

File: data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,537 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+ <teiHeader xml:lang="eng">
+  <fileDesc>
+   <titleStmt>
+    <title xml:lang="lat">Acta Joannis (recenscio)</title>
```
Comment (planatheisa) [inline data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml]: No `<author>`?

File: data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+        </persName>
+       </editor>
+       <author ref="urn:cts:greekLit:tlg0317" xml:lang="lat">Acta Joannis</author>
+       <imprint>
+        <publisher>Mendelssohn</publisher>
+        <pubPlace>Leipzig</pubPlace>
+        <date>1898</date>
+       </imprint>
+       <biblScope unit="volume">2.1</biblScope>
+      </monogr>
```
Comment (planatheisa) [inline data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml]: No scan?

File: data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml (hunk 1 above)
Comment (sonofmun) [reply data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml]: The author is the same as the title of the work, i.e., "Acta Joannis", so I don't think it is necessary.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +35 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +2529 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0317/__cts__.xml`                          | New      |
| `./data/tlg0317/tlg002/__cts__.xml`                   | New      |
| `./data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml` | New      |
| `./data/tlg0317/__cts__.xml`                          | Passing  |
| `./data/tlg0317/tlg002/__cts__.xml`                   | Passing  |
| `./data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3664)*

File: data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml (hunk 1 above)
Comment (planatheisa) [reply data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml]: As with the previuos file, we should add the author here,too.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |      +35 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +2529 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0317/tlg002/__cts__.xml`                   | New      |
| `./data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml` | New      |
| `./data/tlg0317/tlg002/__cts__.xml`                   | Passing  |
| `./data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3811)*

**Pull request merged**